convert_hf_to_gguf.py

```diff
@@ -7928,6 +7928,62 @@ def prepare_tensors(self):
             if len(experts) > 0:
                 raise ValueError(f"Unprocessed experts: {experts}")
 
+
+@ModelBase.register("CogVLMForCausalLM")
+class CogVLMVisionModel(MmprojModel):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.hparams['num_attention_heads'] = self.hparams['num_heads']
+
+    def set_gguf_parameters(self):
```

Comment:
```suggestion
    def set_gguf_parameters(self):
```
Add `num_heads` to the list here instead:
https://github.com/ggml-org/llama.cpp/blob/de221570f9f3ed1bf352516525ea30b0fe73c5a5/convert_hf_to_gguf.py#L1258

Reply:
So I didn't end up doing this, because after I add the "num_heads" to the list, self.find_vparam(["num_attention_heads"]) will still fail due to not finding the key.  
I think this workaround would need to stay... Unless there's some other way?

Reply:
In case it wasn't clear, I meant modifying the list like this: `self.find_vparam(["num_attention_heads", "num_heads"])`

Then it should just work and you don't need to change `hparams`.